src/Cli/dotnet/commands/dotnet-tool/list/ToolListCommandParser.cs

```diff
@@ -23,6 +23,11 @@ internal static class ToolListCommandParser
 
         public static readonly CliOption<string> ToolPathOption = ToolAppliedOption.ToolPathOption;
 
+        public static readonly CliOption<string> ToolListFormatOption = new("--format")
```

Comment:
Instead of a string, we should make a proper enum type here with `Table` and `Json` values - System.CommandLine can bind to these, and we get a nicer help text + validation for that. It also makes the 'switch' statement in the implementation more natural.